extras/check_code_style.sh

```diff
@@ -280,9 +280,9 @@ if $PERFORM_CLANG_FORMAT; then
   fi
 fi
 
-# Verify the PEP8 installation.
+# Verify the pycodestyle installation.
 if $PERFORM_PEP8; then
-  $PEP8 --ignore="E121,E501" ./extras/parse_build_log.py || error_exit "Failed to verify the PEP8 installation. Executable: $PEP8"
+  $PEP8 --ignore="E121,E501,W504" ./extras/parse_build_log.py || error_exit "Failed to verify the pycodestyle installation. Executable: $PEP8"
```

Comment:
I just fixed some of these W504 warnings (line break after binary operator) in a branch of mine and in return got a bunch of W503 (line break *before* binary operator) :disappointed:

As I think the readability of those code locations depends more on context and a general rule will always fail in some cases, I suggest to ignore both: 


```suggestion
  $PEP8 --ignore="E121,E501,W503,W504" ./extras/parse_build_log.py || error_exit "Failed to verify the pycodestyle installation. Executable: $PEP8"
```